src/sentry/static/sentry/app/utils/crashReports.tsx

```diff
@@ -1,6 +1,6 @@
 import {t, tct} from 'app/locale';
 
-export function formatStoreCrashReports(value: number) {
+export function formatStoreCrashReports(value: number | ''): React.ReactNode {
```

Comment:
Won't this output a strange value if `value` is ''?

Reply:
yes, I also find it strange, but I had to add this value to satisfy the compiler ... elsewhere, its value is being used this way..I will find the code and reference it here ;-)

Reply:
here https://github.com/getsentry/sentry/blob/3e83a931e582e0c152cc7f6eaccf147ea7090820/src/sentry/static/sentry/app/views/settings/components/forms/controls/rangeSlider.tsx#L29

Reply:
they are related

Reply:
Interesting, the output of this function with `''` will be a bit funny but not broken.